structs.go

```diff
@@ -1239,13 +1239,51 @@ type Role struct {
 	// This is a combination of bit masks; the presence of a certain permission can
 	// be checked by performing a bitwise AND between this int and the permission.
 	Permissions int64 `json:"permissions,string"`
+
+	// The hash of the role icon. Use Role.IconURL to retrieve the icon's URL.
+	Icon string `json:"icon"`
+
+	// The emoji assigned to this role.
+	UnicodeEmoji string `json:"unicode_emoji"`
+
+	// The flags of the role, which describe extra features of a role.
+	// This is a combination of bit masks; the presence of a certain permission can
+	// be checked by performing a bitwise AND between this int and the flag.
+	Flags RoleFlags `json:"flags"`
 }
 
+// RoleFlags is the flags of "role" (see RoleFlags* consts)
+// https://discord.com/developers/docs/topics/permissions#role-object-role-flags
+type RoleFlags int
+
+// Block containing known RoleFlags values
+const (
+	// RoleFlagInPrompt Role can be selected by members in an onboarding prompt.
```

Comment:
```suggestion
	// RoleFlagInPrompt indicates a Role to be selectable by members in an onboarding prompt.
```

Reply:
I copied the description straight from the discord docs like I usually do. Any reason to deviate here?